docs/source/index.rst

```diff
@@ -8,30 +8,192 @@ ceph-ansible
 Ansible playbooks for Ceph, the distributed filesystem.
 
 
-Testing
-=======
+Installation
+============
 
-* :doc:`Testing with ceph-ansible <testing/index>`
-* :doc:`Glossary <testing/glossary>`
+github
+------
+You can install directly from the source on github by following these steps:
+
+- Clone the repository::
+
+      git clone https://github.com/ceph/ceph-ansible.git
+
+- Next, you must decide which branch of ``ceph-ansible`` you wish to use. There
+  are stable branches to choose from or you could use the master branch::
+
+      git checkout $branch
+
+Releases
+========
+
+The following branches should be used depending on your requirements. The ``stable-*``
+branches have been QE tested and sometimes recieve backport fixes throughout their lifecycle.
+The ``master`` branch should be considered experimental and used with caution.
+
+- ``stable-2.1`` Support for ceph version ``jewel``. This branch supports ansible versions
+  ``2.1`` and ``2.2.1``.
+
+- ``stable-2.2`` Support for ceph versions ``jewel`` and ``kraken``. This branch supports ansible versions
```

Comment:
It really doesn't matter which one you use, it just depends on your requirements. Some might require ansible 2.1 and only deploy jewel so they might use ``stable-2.1``. Others might chose to use ``master`` to get quick fixes and new features. I've just tried to list here what each branch supports because this question has come up often.